simulator-core/src/test/java/com/evbox/everon/ocpp/functional/authorization/EvDriverAuthorizationUsingRfidTest.java

```diff
@@ -0,0 +1,63 @@
+package com.evbox.everon.ocpp.functional.authorization;
+
+import com.evbox.everon.ocpp.simulator.station.actions.Plug;
+import com.evbox.everon.ocpp.testutils.ocpp.exchange.Authorize;
+import com.evbox.everon.ocpp.testutils.ocpp.exchange.StatusNotification;
+import com.evbox.everon.ocpp.testutils.ocpp.exchange.TransactionEvent;
+import com.evbox.everon.ocpp.testutils.station.StationSimulatorSetUp;
+import org.junit.jupiter.api.Test;
+
+import static com.evbox.everon.ocpp.testutils.constants.StationConstants.*;
+import static com.evbox.everon.ocpp.v20.message.common.IdToken.Type.ISO_14443;
+import static com.evbox.everon.ocpp.v20.message.station.StatusNotificationRequest.ConnectorStatus.OCCUPIED;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionData.ChargingState.CHARGING;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionData.ChargingState.EV_DETECTED;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionEventRequest.EventType.STARTED;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionEventRequest.EventType.UPDATED;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionEventRequest.TriggerReason.CABLE_PLUGGED_IN;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionEventRequest.TriggerReason.CHARGING_STATE_CHANGED;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+public class EvDriverAuthorizationUsingRfidTest extends StationSimulatorSetUp {
+
+    @Test
+    void shouldStartChargingWithPreAuthorization() {
+        int seqNo = 0;
+
+        ocppMockServer
+                .when(Authorize.request(DEFAULT_TOKEN_ID, ISO_14443))
```

Comment:
Consider the following pattern: Authroize.request().withToken(...).withIso(...)
as its easier to read